packages/manager/apps/telecom/src/app/telecom/pack/slots/hostedEmail/detail/translations/Messages_fr_FR.json

```diff
@@ -48,5 +48,9 @@
   "hosted_email_detail_client_delete_account_warning": "Attention ! En supprimant votre compte, l'ensemble des messages qu'il contient seront <strong>définitivement</strong> supprimés. Aucune procédure de récupération des données ne sera possible après cette suppression.",
   "hosted_email_detail_client_delete_account_confirm": "Confirmer la suppression",
   "hosted_email_detail_client_delete_account_success": "Votre compte {{email}} est en cours de suppression.",
-  "hosted_email_detail_client_delete_account_error": "Oups ! La suppression du compte a échoué."
-}
\ No newline at end of file
+  "hosted_email_detail_client_delete_account_error": "Oups ! La suppression du compte a échoué.",
+  "input_password_strength":"Robustesse du mot de passe :",
```

Comment:
Please pay attention to the way that this file is formatted.
Missing a space right after the `:` character.

Expected format:
```txt
{
<SPACE><SPACE>"<YOUR_KEY>":<SPACE>"<YOUR_VALUE>"
}
```